micro-rdk-nmea/src/gen/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod enums;
```

Comment:
should this file also be auto-generated?
If it's here but the enums.rs generation get skipped because there was no json, does this cause issues?